src/components/Input/Input.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function Input({ labelText, placeholder, name }) {
+  return (
+    <Label className="Input">
```

Comment:
Here I don't understand why are you using a className if your are using styled-components to do the styling